terasoluna-gfw-functionaltest-web/src/main/webapp/WEB-INF/views/pagination/search.jsp

```diff
@@ -2,76 +2,72 @@
 <h1 class="center-content">Person Search Form</h1>
 
 <form class="center-content">
-    <spring:nestedPath path="personSearchForm">
-        <form:input path="name" />
-        <form:button id="searchButton">Search</form:button>
-    </spring:nestedPath>
+	<spring:nestedPath path="personSearchForm">

```

Comment:
@ikeyat 
This changes to be expected to escape `ParseException` when new line exists in expression.
But I've to understand that there is a following of such a solution, I removed These unneccessary changes.

```xml
<jsp-attribute name="doNotEscapeWhitespaceCharsInExpression" value="true" />
```